.github/workflows/pr-build.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches: [master]
   pull_request:
-    branches: [master]
+    branches: [master, aws-v3-support]
```

Comment:
@willarmiros Can you switch this PR to draft?

Reply:
Ah good call, sorry for the spam